packages/blocks/src/text-input/validations.js

```diff
@@ -1,11 +1,26 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 export default {
 	emailValidation: ( value ) => {
 		if (
 			! value.match(
 				/^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
 			)
 		) {
-			return 'Please enter a valid email address.';
+			return __( 'Please enter a valid email address.' );
```

Comment:
Translation calls should also carry a domain (second argument). I think we're matching domains to the package names, so `blocks` should the domain here.